brain-marks/Settings/Views/WhatsNewView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  WhatsNewView.swift
+//  brain-marks
+//
+//  Created by Susannah Skyer Gupta on 10/25/22.
+//
+
+import SwiftUI
+
+struct WhatsNewView: View {
+    var body: some View {
+      VStack(alignment: .leading, spacing: 10) {
+          Text("**v1.2**")
+          .font(.title3)
+          Text("* Delightful updates arriving soon!")
+          Text("**v1.1**")
+          .font(.title3)
+          Text("* Fix add folder and add tweet button not working in iOS 15")
+      }
+      .padding()
+      Spacer()
+    }
+}
```

Comment:
Instead, I was looking for you to create a `.md`, markdown file, and read from that in this view, rather than hard code the changes into the `View` itself

Also I'd recommend putting it into a `ScrollView` to accommodate larger dynamic type sizes